### PR TITLE
Cryptopanic

### PR DESCRIPTION
**Pull request by JakubPluta**
Hey there!

I found very cool news aggregator for crypto - Cryptopanic. They provide crypto news from many different sources, with filtering features (by regions, kind of news etc). The API is free, you only need to create your API key.

https://cryptopanic.com/developers/api/

I put it for now in overview controller. Maybe we should aim for separate menu for all kind of news, twitter, reddit features ? 
Let me know what do you think :)

### CODE REVIEW

**Review by piiq — APPROVED** [review]
That's a great recourse you found there and the news filtering is nice

File: gamestonk_terminal/cryptocurrency/overview/overview_controller.py [hunk 1]
```diff
@@ -1529,6 +1534,116 @@ def call_cbpairs(self, other_args):
...
+            "--links",
+            dest="links",
+            action="store_true",
+            help="Flag to show urls. If you will use that flag you will additional column with urls",
+            default=False,
+        )
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_BOTH_RAW_DATA_AND_FIGURES
+        )
```
Comment (jmaslek) [inline gamestonk_terminal/cryptocurrency/overview/overview_controller.py:1631]: Should just be EXPORT_RAW_DATA ...or whichever is that one.

**Comment by jmaslek** [comment]
Noiceee.  Lat thing is to update the Hugo main.yml :)

File: gamestonk_terminal/cryptocurrency/overview/overview_controller.py (hunk 1 above)
Comment (JakubPluta) [reply gamestonk_terminal/cryptocurrency/overview/overview_controller.py:1631]: Will fix it :)

**Comment by JakubPluta** [comment]
> Noiceee. Lat thing is to update the Hugo main.yml :)

Could you point me to that main.yml file? I don't know in which place it's stored :D

**Comment by jmaslek** [comment]
> Could you point me to that main.yml file? I don't know in which place it's stored :D

Website/data/main.yml

**Review by jmaslek — APPROVED** [review]

**Pull request merged**